components/filter/filter.js

```diff
@@ -917,6 +951,13 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 		subscribers.forEach(subscriber => subscriber.updateActiveFilters(this.id, this._activeFilters));
 	}
 
+	_updateDimensionShouldBubble(dimension) {
+		for (const value of dimension.values) {
+			if (value.selected) value.shouldBubble = true;
+			else value.shouldBubble = false;
```

Comment:
Minor: This could technically be something like `value.shouldBubble = !!value.selected;`, but I don't mind it like this either!

Reply:
Yep, I don't think it needs to use `!!` since `selected` is already a boolean.